src/Http/Routing/src/Constraints/RegexErrorStubRouteConstraint.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Routing.Constraints;
+
+internal sealed class RegexErrorStubRouteConstraint : IRouteConstraint
+{
+    public RegexErrorStubRouteConstraint([StringSyntax(StringSyntaxAttribute.Regex, RegexOptions.CultureInvariant | RegexOptions.IgnoreCase)] string _)
```

Comment:
```suggestion
    public RegexErrorStubRouteConstraint(string _)
```

This is not necessary.